src/main/java/com/codeit/duckhu/domain/comment/service/CommentService.java

```diff
@@ -0,0 +1,84 @@
+package com.codeit.duckhu.domain.comment.service;
+
+
+import com.codeit.duckhu.domain.comment.exception.NoCommentException;
+import com.codeit.duckhu.domain.comment.domain.Comment;
+import com.codeit.duckhu.domain.comment.dto.CommentDto;
+import com.codeit.duckhu.domain.comment.repository.CommentRepository;
+import com.codeit.duckhu.domain.comment.dto.request.CommentCreateRequest;
+import com.codeit.duckhu.domain.comment.dto.request.CommentUpdateRequest;
+import com.codeit.duckhu.domain.user.service.UserServiceImpl;
+import com.codeit.duckhu.review.service.impl.ReviewServiceImpl;
+import java.util.UUID;
+import lombok.RequiredArgsConstructor;
+import org.springframework.data.domain.Slice;
+import org.springframework.stereotype.Service;
+
+@Service
+@RequiredArgsConstructor
+public class CommentService {
+  private final CommentRepository repository;
+  private final CommentMapper commentMapper;
+
+  private final UserServiceImpl userService;
+  private final ReviewServiceImpl reviewService;
+
+   public CommentDto get(UUID id){
+     if(repository.findById(id).isEmpty()){
+       throw new NoCommentException(ErrorCode.NOT_FOUND_COMMENT);
+     }
+
+     Comment comment = repository.findById(id).get();
+
+     return commentMapper.toDto(comment);
+   }
+
+   public Slice<CommentDto> getList(){
+
+   }
+
+   //TODO : User & Review service 이용하여 객체 불러오기 필요 : 현재 메서드 미구현 상태
+   public CommentDto create(CommentCreateRequest request){
+     Comment comment = Comment.builder()
+         .user()
+         .review()
+         .content(request.getContent())
+         .build();
+
+     repository.save(comment);
+
+     return commentMapper.toDto(comment);
+   }
+
+   public void delete(UUID id){
+     if(repository.findById(id).isEmpty()){
```

Comment:
```suggestion
     if(!repository.existsById(id)){
```
지금 코드로도 문제 없이 동작하긴 하지만, 이 부분은 단순히 존재 여부만 확인하는 로직이라  
`findById()` 대신 `existsById()`를 사용하는 게 의도에 대해 더 명확하게 알 수 있을 것 같아
추후 리팩토링 방향으로 고려해보셔도 좋을 것 같습니다!